Final update to the Iron complete changelog.

clalancette opened the pull request.

clalancette committed:
Final update to the Iron complete changelog.

Signed-off-by: Chris Lalancette <clalancette@gmail.com>

Yadunund reviewed (commented):
Some of the changelogs have `mergify[bot]` appended as a contributor. The same ones also link two PRs in the message- the original and the backport. Do we want to clean these up?

clalancette commented:
> Some of the changelogs have `mergify[bot]` appended as a contributor. The same ones also link two PRs in the message- the original and the backport. Do we want to clean these up?

It's a bit unfortunate, but I'd prefer not to.  The main reason is that my scripts just blindly go and get the contributors out of the git logs, and technically the author of those commits is indeed `mergify[bot]`.  If we start modifying things by hand here, then we risk "losing" those modifications if we have to run the scripts for some reason again.

Yadunund commented on source/Releases/Iron-Irwini-Complete-Changelog.rst:
```diff
@@ -1553,6 +1556,8 @@ This page is a list of the complete changes in all ROS 2 core packages since the
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+* Fix ClassLoader warning by unloading plugin providers. (`#275 <https://github.com/ros-visualization/qt_gui_core/issues/275>`__)
+* Chen Lihui
```
Move to contributors list below

Yadunund replied:
Ah sorry @clalancette I reviewed this before seeing your comment. Would this modification also pose the same risk?

clalancette replied:
Yes, but this is also a bit weird.  Let me take a look at the script and see if there is a bug in it.

clalancette replied:
Oh, right, I see.  The changelog itself is a bit wonky: https://github.com/ros-visualization/qt_gui_core/blob/4abf0eebc7a0da73f00c1c8432166efb16e244b1/qt_gui_cpp/CHANGELOG.rst?plain=1#L8 .  That line should say "Contributors: Chen Lihui", which is why my script isn't picking it up properly.

So yeah, in that case I'd prefer not to change this for the same reason; the script wouldn't have generated that.

clalancette replied:
All of that said, this did cause me to go back and re-examine the code in https://github.com/clalancette/ros_changelog_from_pkgs that I use to generate this.  And I noticed that we were missing the changelog for a few new packages.  I've now fixed the generator and updated this PR with the missing data.

Yadunund replied:
Whoops that changelog error is totally on me- Sorry 🤦🏼 
Thanks for all the fixes!

clalancette committed:
Update from fixes to the generator.

After looking at the generator again, I noticed that it
was actually missing the changelog for some new packages.
I ended up fixing the generator, and this commit is the
result.

Signed-off-by: Chris Lalancette <clalancette@gmail.com>
